slo_generator/backends/prometheus.py

```diff
@@ -153,7 +164,7 @@ def query(self, filter, window, timestamp=None, operators=[], labels={}):
         return response
 
     @staticmethod
-    def count(response):
+    def count(response: dict) -> float:
```

Comment:
Are you sure about the `float` return type?

Reply:
Yeah sure check line 177

Reply:
Agreed, but then it makes little sense to call the function `count`, document the return type as `int` in the docstring and return `NO_DATA` when an exception is raised. I am pretty sure we should replace `float` with `int` everywhere. Let me ask Olivier who wrote this code in the first place.

Reply:
Yeah right I got confused the first time checking this function, the docstring says it returns `int` but on return `line 177` there's `float`.

I'll wait Olivier's response 👀

Reply:
same as above, we need to observe some Prometheus responses and use it as-is I believe.

Reply:
Same here. Let's keep `float` until we know better.